ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -95,7 +95,7 @@ def _compile_tag_re(self):
     def tags(self):
         return [
             "queue_manager:{}".format(self.queue_manager_name),
-            "host:{}".format(self.host),
+            "mq_host:{}".format(self.host),  # host tag already used for agent host
```

Comment:
I find the comment a bit confusing, if we want to leave a trail of why we picked `mq_host` we could say something like `# 'host' is reserved and 'mq_host' is used instead`